source/includes/api/_layer.md

```diff
@@ -807,3 +807,291 @@ userName                | String         | No (autogenerated)   | null
+
+Updating contextual layers also has some caveats:
+
+- you can only update the `enabled` attribute of contextual layers you own, or contextual layers that belong to a team that you manage - if one of these conditions is not met, the `enabled` attribute value is kept as is.
+- you can only update the `isPublic` attribute of contextual layers if you are an `ADMIN` user and the contextual layer you are trying to update is owned by a user (not by a team) - in all other cases, the `isPublic` attribute value is kept as is.
+
+#### Errors for updating a layer
+
+Error code     | Error message           | Description
+-------------- | ----------------------- | ---------------------------------------------------------------------------------
+404            | Layer not found         | No contextual layer was found for the provided id.
+500            | Layer retrieval failed. | An error occurred while fetching the layer information from the database.
+500            | Team retrieval Failed.  | An error occurred while fetching the information about the team with id provided.
+500            | Layer update failed.    | An error occurred while saving the updated information of the contextual layer.
+
+### Deleting a contextual layer
+
+> Example DELETE request to delete a contextual layer:
+
+```shell
+curl -X DELETE "https://api.resourcewatch.org/v1/contextual-layer/:layerId" \
+-H "Authorization: Bearer <your-token>" \
+-H "Content-Type: application/json"
+```
+
+Use this endpoint if you'd like to delete a contextual layer from the RW API. In order to delete a contextual layer, you must either be an `ADMIN` user or, alternatively, meet all of the following conditions:
+
+- the contextual layer's `isPublic` property must be set to `false`.
+- if the contextual layer is owned by a user, you must be the owner of the contextual layer.
+- if the contextual layer is owned by a team, you must be one of the managers of the team that owns the contextual layer.
+
+If successful, the request to delete a contextual layer returns a 204 No Content response.
+
+#### Errors for deleting a layer
+
+Error code     | Error message           | Description
+-------------- | ----------------------- | ---------------------------------------------------------------------------------
+404            | Layer not found         | No contextual layer was found for the provided id.
+500            | Layer retrieval failed. | An error occurred while fetching the layer information from the database.
+500            | Team retrieval Failed.  | An error occurred while fetching the information about the team with id provided.
+500            | Layer update failed.    | An error occurred while saving the updated information of the contextual layer.
+
+### Getting loss layer tiles
```

Comment:
Not sure I understand what this is. Is this an endpoint that references a particular group of contextual layers, or are loss layers some kind of transformation of contextual layers?

My guess is it's the first one, but I would just add a sentence or two just saying that this is a special endpoint that refers to a particular set of commonly used contextual layers (like we did with the geostores).

Reply:
From what I understood from the source code, this endpoint works like a shortcut to get PNG tiles - instead of retrieving the `url` field of the contextual layer and manually interpolating values in users' applications, as users would have to do for "normal" contextual layers.

It only works for getting tiles for a URL configured in the service's env variables - I don't think there's even  relationship with contextual layers at all... So not really sure how to phrase this, but I agree this calls for some clarification, so I added a paragraph trying to clarify the behavior of this endpoint - let me know your thoughts :)

Reply:
Haha okay I guess that's good for now!